src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -170,6 +170,8 @@ private static String injectRealAccounts(String jsonString) {
                 .replace("${test.student1}", TestProperties.inst().TEST_STUDENT1_ACCOUNT)
                 .replace("${test.student2}", TestProperties.inst().TEST_STUDENT2_ACCOUNT)
                 .replace("${test.instructor}", TestProperties.inst().TEST_INSTRUCTOR_ACCOUNT)
+                .replace("${test.unreg}", TestProperties.inst().TEST_UNREG_ACCOUNT)
+                .replace("${test.admin}", TestProperties.inst().TEST_ADMIN_ACCOUNT)
```

Comment:
The tests passed because we don't have any test case that used the test.admin account in the json files.